controller/Review.php

```diff
@@ -152,7 +152,9 @@ public function getItem(): void
 
             // make sure the responses data are compliant to QTI definition
             $itemData['data']['responses'] = array_filter($responsesData, static function ($key) use ($responsesData) {
```

Comment:
It looks like it can be replaced with a more IMO readable piece of code.
```
array_filter(
    $responsesData,
    static function (array $responseEntry): bool {
        return array_key_exists('qtiClass', $responseEntry)
            && array_key_exists('serial', $responseEntry)
            && $responseEntry['qtiClass'] !== 'modalFeedback';
    }
);
```